draft-ietf-quic-transport.md

```diff
@@ -1589,9 +1589,13 @@ Receiving a PATH_CHALLENGE frame from a peer indicates that the peer is probing
 for reachability on a path. An endpoint sends a PATH_RESPONSE in response as per
 {{migrate-validate}}.
 
-PATH_CHALLENGE, PATH_RESPONSE, and PADDING frames are "probing" frames. Until an
-endpoint receives packets containing other frames from a given peer address, it
-MUST NOT send other types of frames to that address.
+PATH_CHALLENGE, PATH_RESPONSE, and PADDING frames are "probing frames", and all
+other frames are "non-probing frames".  A packet containing only probing frames
+is a "probing packet", and a packet containing any other frame is a "non-probing
+packet".
+
+An endpoint MUST NOT send non-probing frames to a peer address until it receives
```

Comment:
Is this statement true?  What about NAT rebinds?

Reply:
If this statement is strictly followed, no one ever migrates because they're waiting for the other side to do so.  (At which the other side has violated a MUST, and you really ought to kill the connection to that naughty endpoint!)

For both client and server migration, this actually points in the same direction, so let's accept the non-generic here:  "A server MUST NOT send non-probing packets to a client address until it receives a non-probing packet from that address."